### DISCUSSION TITLE
What's the best approach to create an algorithm that predicts which text posts a user might like (like on Twitter) ?

### DISCUSSION THREAD

**Discussion by FabianK-Dev** [BrainJS/brain.js#742] [Q&A] [answered]
First of all: Thanks for the amazing library! It's incredible to work with it!

Let's say you have three different lists of text posts:
- 1. a list of posts that the user liked
- 2. a list of posts that the user saw but didn't like (= disliked)
- 3. a list of posts that the user neither saw nor liked

What's the best approach to take the data from 1. and 2. to learn which kind of posts a user likes/dislikes and then pick a post from 3. based on what the algorithm learned about the user's interests?

To keep it simple I would like to only focus on posts containing text. The algorithm should only use the post's text content and whether the post was liked or not liked by the user as training data to predict which other post the user might like.

Thanks!

**Comment by robertleeplummerjr** [comment, marked as answer]
`brain.recurrent.LSTM`
